uefi/src/proto/device_path/mod.rs

```diff
@@ -377,6 +424,51 @@ impl DevicePath {
         let data = data.into_boxed_slice();
         unsafe { mem::transmute(data) }
     }
+
+    /// Variant of [`Self::to_string`] that creates a list of tuples of each
+    /// [`DevicePathNode`] and the corresponding text representation using the
+    /// [`DevicePathToText`] protocol.
+    ///
+    /// The resulting string is only None, if there was not enough memory.
+    #[cfg(feature = "alloc")]
+    pub fn to_node_string_pairs(
```

Comment:
Is this interface something that would be commonly needed? It could be done easily with some iterator chaining (`path.node_iter().map(...).collect()`), so I don't think we need to provide a function for it unless there's a reason why it would be used a lot.

Reply:
yes, you are right. This is not needed anymore. Good cathc!